cmd/generate-index/provider.yaml

```diff
@@ -18,6 +16,8 @@ cdn:
       - https://api.fastly.com/public-ip-list
     google:
       - https://www.gstatic.com/ipranges/goog.json
+    stackpath:
+      - https://k3t9x2h3.map2.ssl.hwcdn.net/ipblocks.txt
```

Comment:
@neonbunny do we know the source of this url / hosted by?

Reply:
It's from the stackpath support pages - see comment on commit 4783aa5e71636c84b780c159ee206d0625f52db4